html/includes/print-map.inc.php

```diff
@@ -51,7 +60,7 @@
             }
             if ($v['id'] == $link_devices['remote_device_id']) {
                 $to = $v['id'];
-                $port .= ' > ' .shorten_interface_type($link_devices['remote_port']);
+                $port = ' > ' .shorten_interface_type($link_devices['remote_port']);
```

Comment:
This is overwriting a previously set value of $port. This is supposed to build up a string that says interface1 > interface2.